Framework/DataObjects/inc/MantidDataObjects/EventList.h

```diff
@@ -456,6 +456,8 @@ class MANTID_DATAOBJECTS_DLL EventList : public Mantid::API::IEventList {
                                 Mantid::Kernel::Unit *toUnit);
   template <class T>
   void convertUnitsQuicklyHelper(typename std::vector<T> &events, const double &factor, const double &power);
+
+  template <class T> void copy_if(std::vector<T> &events, std::vector<T> &output, bool (*pf)());
```

Comment:
since we are no longer creating our own copy_if template, shouldn't this declaration be removed?

Reply:
Yes, I removed it from the .cpp missed it in the .h